server.cpp

```diff
@@ -27,10 +31,19 @@ std::string make_daytime_string()
     return buffer;
 }
 
+size_t read_complete(char * buff, const error_code & err, size_t bytes)
```

Comment:
In other places you don't put a whitespace betwee a type and a '*' or '&'. Be consistent.